libraries/AP_NavEKF2/AP_NavEKF2_PosVelFusion.cpp

```diff
@@ -315,9 +315,6 @@ void NavEKF2_core::SelectVelPosFusion()
     // Check for data at the fusion time horizon
     extNavDataToFuse = storedExtNav.recall(extNavDataDelayed, imuDataDelayed.time_ms);
 
-    // read GPS data from the sensor and check for new data in the buffer
```

Comment:
why does this PR change the way we recall GPS samples?

Reply:
Because we need GPS data available to the yaw estimator which in turn needs to be calculated early in the frame.